api/src/utils/env.ts

```diff
@@ -34,7 +34,8 @@ assert.ok(process.env.FCC_ENABLE_DEV_LOGIN_MODE);
 if (process.env.FREECODECAMP_NODE_ENV !== 'development') {
   assert.ok(process.env.PORT);
   assert.ok(process.env.MONGOHQ_URL);
-  assert.ok(process.env.SENTRY_DSN);
+  if (process.env.SENTRY_DSN !== 'dsn_from_sentry_dashboard')
+    assert.ok(process.env.SENTRY_DSN);
```

Comment:
Hang on a second.

All this is doing is asserting that:
- We're not in development ENV
- Sentry DSN is not `dsn_from_sentry_dashboard`
- Sentry DSN is not falsy

But there's no conditional returns in the export block down below? So if the DSN *is* `dsn_from_sentry_dashboard`, it'll be exported as that and will trigger the error when we attempt to mount the middleware.

IMHO, the export on line `63` should be updated to:

```js
export const SENTRY_DSN = process.env.SENTRY_DSN === "dsn_from_sentry_dashboard" ? "" : process.env.SENTRY_DSN
```

Or some similar logic.